### DISCUSSION TITLE
ConceptTensorFlowObjectDetetion:Camera recognition failed

### DISCUSSION THREAD

**Discussion by lovezfy** [FIRST-Tech-Challenge/FtcRobotController#847] [Q&A] [answered]
FTC APP 9.0.1.  Run ConceptTensorFlowObjectDetetion. But this camera is slow to recognize pixel. In a minute, I might get one successful recognition, and it's not good. Do you have similar problems? Can you give me some advice?

**Comment by texasdiaz** [comment, marked as answer]
It's absolutely correct that given the processing power of the control hub (assuming you're using a control hub) and the processor intensive nature of inference models, instead of measuring frames per second you might as well measure seconds per frame (or so it seems sometimes). However, more than a few seconds is extreme. I'm wondering if you are trying to recognize the pixel at too extreme of an angle - with the default TensorFlow model for 2023 and the unique shape of the pixel, you cannot detect the pixel from the starting location.

Please be sure to read the following article:

* https://ftc-docs.firstinspires.org/programming_resources/vision/tensorflow_cs_2023/tensorflow-cs-2023.html

-Danny
